Makefile

```diff
@@ -23,27 +23,28 @@ install:
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 	echo "Installation successful (Mac OS)"
 uninstall:
-	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
+	rm $(DESTDIR)$(PREFIX)/bin/ani-cli
 	echo "Removal successful (Mac OS)"
 
 else ifeq ($(Platform), Android)
-TERMUX_BIN := /data/data/com.termux/files/usr/bin
```

Comment:
This one makes me worry. We should probably assign PREFIX to that, what do you think?

Reply:
PREFIX := /data/data/com.termux/files/usr/bin

Reply:
Well current master doesn't use the variable at all, but that could just mean that that got broken along the way too

Reply:
Termux already set PREFIX to /data/data/com.termux/files/usr
so adding PREFIX is useless i think

Reply:
> Termux already set PREFIX to /data/data/com.termux/files/usr so adding PREFIX is useless i think

wdym Termux set it? As in a default Termux environment has that variable by default?

Reply:
i tested it and it works with or without it

Reply:
then we can leave it out

Reply:
> wdym Termux set it? As in a default Termux environment has that variable by default?

yup termux has that prefix by default.. and removing above line won't do any harm... I think this PR is good to go.

![image](https://user-images.githubusercontent.com/76195824/150976957-c838fc8b-6200-4fe1-a1f5-3de3deadbb6d.png)